src/analytics/Properties.tsx

```diff
@@ -1203,6 +1203,14 @@ type SwapQuoteEvent = SwapEvent & {
   provider: string
 }
 
+export interface SwapTimeMetrics {
```

Comment:
Some nits:
- the names aren't completely descriptive.  since you explained them in the commit body, it might be worth adding those explanations as comments
- I'm struggling with the "elapsed" times.  Sometimes they mean "time to make a specific call within this larger swap operation" and sometimes they mean "elapsed since the start of the swap operation".  Can we be consistent?  I'd suggest the latter (and then encourage us to use a profiling/APM tool for the former).